### PR TITLE
Log Installer File Verification

### PR DESCRIPTION
**Pull request by nagilson**
This will improve our ability to repudiate and view logs when failures with sign checking occur

### CODE REVIEW

File: vscode-dotnet-runtime-library/src/Acquisition/GlobalInstallerResolver.ts [hunk 1]
```diff
@@ -249,6 +250,10 @@ Please report this issue so it can be remedied or investigated.`));
                             this.eventStream.post(releaseJsonErr);
                             throw releaseJsonErr.error;
                         }
+                        else
+                        {
+                            this.eventStream.post(new DotnetFileIntegrityCheckEvent(`This installer file is hosted on an expected domain https://download.visualstudio.microsoft.com/.`));
```
Comment (JL03-Yue) [inline vscode-dotnet-runtime-library/src/Acquisition/GlobalInstallerResolver.ts:255]: https://download.visualstudio.microsoft.com/ Does it redirect to an empty page?

**Review by JL03-Yue — APPROVED** [review]
Looks good to me

File: vscode-dotnet-runtime-library/src/Acquisition/GlobalInstallerResolver.ts (hunk 1 above)
Comment (nagilson) [reply vscode-dotnet-runtime-library/src/Acquisition/GlobalInstallerResolver.ts:255]: That's a good thing to point out. This is in the log so it should be fine. I am a bit surprised its empty, but it is just a download server, so kinda makes sense the root is 404

**Pull request merged**